modular_skyrat/modules/aesthetics/lights/code/lighting.dm

```diff
@@ -6,8 +6,6 @@
 	light_power = 1.25
 
 /obj/machinery/light
-	icon = 'modular_skyrat/modules/aesthetics/lights/icons/lighting.dmi'
```

Comment:
Any references to icon state "tube_lpower" in modular code needs to be changed to "tube_emergency"

Reply:
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/83487515/7d21f67c-7f82-410e-ab11-e46091b826c2)

Have you tested other icon states?

Reply:
Shiit I seen it, we had an entirely unique state for it that was labelled "Skyrat Edit" with no tie in to Lighting or Aesthetics 💀 I'll go through and test 'em all now

Reply:
Tested them all, for both tubes and bulbs. 
On, Broken, Burnt, No Power, Off
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/76465278/42cff379-d398-4ad5-9759-1b7b9f9e0d6d)